include/motor-control/core/tasks/tmc2130_motor_driver_task.hpp

```diff
@@ -86,6 +86,15 @@ class MotorDriverMessageHandler {
         }
     }
 
+    void handle(const can::messages::ReadMotorDriverErrorStatus& m) {
```

Comment:
this class's spi response handler needs to look at the tag like in the 2160 driver right